test_unit.py

```diff
@@ -36,3 +35,45 @@ def test_host_fips_disabled(tmp_path):
     fipsfile = tmp_path / "fips"
     fipsfile.write_text("")
     assert not host_fips_enabled(f"{fipsfile}")
+
+
+def test_routing_keys():
+    con = RabbitMQConnection(message_topic_prefix="foo.bar")
```

Comment:
isn't there a risk of failing unit tests with there are problems to reach opensuse? 
I am wondering if we should mock here, instead.

Reply:
No, this unit test does not connect to the bus at all.